fdbclient/StorageServerInterface.h

```diff
@@ -212,14 +213,23 @@ struct GetValueRequest : TimedRequest {
 	Optional<TagSet> tags;
 	Optional<UID> debugID;
 	ReplyPromise<GetValueReply> reply;
+	VersionVector ssLatestCommitVersions; // includes the latest commit versions, as known
+	                                      // to this client, of all storage replicas that
+	                                      // serve the given key
```

Comment:
Propagating the latest commit versions of all replicas here. This way the receiving storage server will get its latest commit version irrespective of the replica that the load balancer chooses to send the message to. Since the number of storage server replicas for a given shard is not high, this shouldn't add a significant amount of overhead. Let me know if it is preferable to modify the load balancer code over this.

Note: We will need to make sure that the latest commit versions of storage servers stay up-to-date (in the version vector on the sequencer) in the presence of key range splits/merges. Not addressing that issue in this PR.

Reply:
>Note: We will need to make sure that the latest commit versions of storage servers stay up-to-date (in the version vector on the sequencer) in the presence of key range splits/merges. Not addressing that issue in this PR.

Can you add a TODO to the code

Reply:
Done (placed that in masterserver.actor.cpp).